src/components/Header/Header.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { ErrorsMessages } from '../../types/ErrorsMessages';
+
+type Props = {
+  todos: Todo[],
+  addTodo: (newTodoTitle: string) => Promise<void>,
+  switchBtn: (AllChecked: boolean) => void
```

Comment:
what is switchBtn? make the name with action part like 'onSwitch' or smth like that